dashboard/pages/Artists.py

```diff
@@ -0,0 +1,29 @@
+"""A dashboard page that lets you view sales by track vs. albums for artists."""
+
+import streamlit as st
+
+from database import get_connection, get_popular_artists, get_all_artists, get_sales, get_sales_for_chosen_artists
+from charts import get_artist_album_sales_bar_chart, get_artist_track_sales_bar_chart, get_most_popular_artists_chart
+
+if __name__ == "__main__":
+
+    conn = get_connection()
+    pop_artists = get_popular_artists(conn)
+    st.altair_chart(get_most_popular_artists_chart(
+        pop_artists), use_container_width=True)
+
+    all_artists = get_all_artists(conn)
+    chosen_artists = st.multiselect(
+        "Choose artists to compare their track vs. album sales", all_artists, placeholder="Choose artists...")
+    all_sales = get_sales(conn)
+    chosen_artists_data = get_sales_for_chosen_artists(
+        all_sales, chosen_artists)
```

Comment:
What is displayed when the user hasn't chosen any artists? Is it an empty chart?

Reply:
Good point, it is currently empty graphs. Let me add an if statement so that it only displayed once artists are chosen.